HACKING.md

```diff
@@ -42,97 +42,60 @@ oc scale --replicas=0 deployment.apps/local-storage-operator -n openshift-local-
 All the images including operator, diskmaker, bundle and index images can be created in one shot using following command:
 
 ```
-~> ./hack/sync_bundle -o <operator_image> -d <diskmaker_image> -b <bundle_image> -i <index_image> bundle
+make bundle REGISTRY=quay.io/username
 ```
 
-This command also pushes the images to selected docker registry, so if you ran this command with following arguments:
+You may optionally set `REPO` to override the repository name, `VERSION` to be used as part of the tag, and `TOOL_BIN` to specify which tool to use (defaults to podman or docker).
 
 ```
-~> ./hack/sync_bundle -o quay.io/gnufied/local-storage-operator:latest  \
-        -d quay.io/gnufied/local-storage-diskmaker:latest \
-        -b quay.io/gnufied/local-storage-bundle:v1 \
-        -i quay.io/gnufied/gnufied-index:v1 bundle
+make bundle REGISTRY=quay.io/username REPO=local-storage-operator VERSION=latest TOOL_BIN=`which docker`
 ```
 
-The command will create all of these images and push them to quay. Operator and diskmaker arguments to `sync_bundle` script can be skipped and in that case `quay.io/openshift/origin-local-storage-diskmaker:latest` and `quay.io/openshift/origin-local-storage-operator:latest` version of those images are used:
-
+This command also pushes the images to selected docker registry. So the command above will push the following images to quay:
 
 ```
-~> ./hack/sync_bundle -b quay.io/gnufied/local-storage-bundle:v1 \
-        -i quay.io/gnufied/gnufied-index:v1 bundle
+quay.io/username/local-storage-operator:operator-latest
+quay.io/username/local-storage-operator:diskmaker-latest
+quay.io/username/local-storage-operator:mustgather-latest
+quay.io/username/local-storage-operator:bundle-latest
+quay.io/username/local-storage-operator:index-latest
 ```
 
-This should give us index image `quay.io/gnufied/gnufied-index:v1`. Update the `CatalogSource` entry in `examples/olm/catalog-create-subscribe.yaml`
-to point to your newly created index image. Once updated, we can install local-storage-operator via following command:
+After pushing all the images, it will print a subscription you can copy and paste to install the catalog source on your cluster:
 
 ```
-~> oc create -f examples/olm/catalog-create-subscribe.yaml
-```
-
-## Manual creation of bundle and index image.
-
-1. Since we will be going to test with our version of images, we need to modify CSV file to point to our version of image. This can be done by modifying following file:
+Copy following snipped to apply it to your cluster
```

Comment:
```suggestion
Copy following snippet to apply it to your cluster
```

Reply:
fixed (here and in hack/create-bundle.sh)